src/Framework/Attributes/RequiresPhpunitExtension.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\Framework\Attributes;
+
+use Attribute;
+use PHPUnit\Runner\Extension\Extension;
+
+/**
+ * @immutable
+ *
+ * @no-named-arguments Parameter names are not covered by the backward compatibility promise for PHPUnit
+ */
+#[Attribute(Attribute::TARGET_CLASS | Attribute::TARGET_METHOD)]
```

Comment:
To be consistent with existing attributes, please change this attribute to be repeatable (and only hold the name of a single extension bootstrapper class instead of an array of class names).

Reply:
yeah sorry, I have not seen that other requirements were repeatable. fixed.